app/src/main/java/org/sopt/and/ui/ContextExt.kt

```diff
@@ -0,0 +1,31 @@
+package org.sopt.and.ui
+
+import android.content.Context
+import android.widget.Toast
+import androidx.annotation.StringRes
+import androidx.compose.material3.SnackbarDuration
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.material3.SnackbarResult
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+fun Context.toast(@StringRes message: Int) {
+    Toast.makeText(this, getString(message), Toast.LENGTH_SHORT).show()
+}
+
+
+fun SnackbarHostState.ShowSnackBar(
```

Comment:
```suggestion
suspend fun SnackbarHostState.showSnackBar(
```
로 하시면 CoroutineScope 받지 않아도 됩니다.

Reply:
헉 감사합니다!!